addons/l10n_syscohada/data/account.account.template.csv

```diff
@@ -35,35 +35,23 @@
-"pcg_132","MARGE COMMERCIALE (MC)","132","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_133","VALEUR AJOUTÉE (V.A.)","133","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_134","EXCÉDENT BRUT D'EXPLOITATION (E.B.E.)","134","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_135","RÉSULTAT D'EXPLOITATION (R.E.)","135","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_136","RÉSULTAT FINANCIER (R.F.)","136","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_137","RÉSULTAT DES ACTIVITÉS ORDINAIRES (R.A.O.)","137","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1381","Résultat de fusion","1381","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1382","Résultat d'apport partiel d'actif","1382","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1383","Résultat de scission","1383","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1384","Résultat de liquidation","1384","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_139","RÉSULTAT NET : PERTE","139","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1411","État","1411","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1412","Régions","1412","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1413","Départements","1413","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1414","Communes et collectivités publiques décentralisées","1414","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1415","Entités publiques ou mixtes","1415","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1416","Entités et organismes privés","1416","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1417","Organismes internationaux","1417","equity","l10n_syscohada.syscohada_chart_template","False"
-"pcg_1418","Autres","1418","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1411","Subvention d'equipement: État","1411","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1412","Subvention d'equipement: Régions","1412","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1413","Subvention d'equipement: Départements","1413","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1414","Subvention d'equipement: Communes et collectivités publiques décentralisées","1414","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1415","Subvention d'equipement: Entités publiques ou mixtes","1415","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1416","Subvention d'equipement: Entités et organismes privés","1416","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1417","Subvention d'equipement: Organismes internationaux","1417","equity","l10n_syscohada.syscohada_chart_template","False"
+"pcg_1418","Subvention d'equipement: Autres","1418","equity","l10n_syscohada.syscohada_chart_template","False"
```

Comment:
This is different from the original PR

Reply:
What is different from the original is that i've change some misclassified account, i have a pr to change that in V15 that wait this one to be merged, for those account i will take a look but i don't think i've changed this ones